### PR TITLE
Factory now returns the modified instance of tv4

### PR DESCRIPTION
**Pull request by GoogilyBoogily**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,9 +63,9 @@ First, require tv4.
 
     tv4 = require("tv4");
 
-Then, require the tv4-file-loader module and pass the newly created tv4 into it.
+Then, require the tv4-file-loader module, pass the newly created tv4 into it, and set the original tv4 to the modified one.
```
Comment (quantumew) [inline README.md]: Can you fix this comment per our discussion? Maybe just say something about why return it. For convenience or something.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+1.0.2 - 2017-02-22
+------------------
+
+* The factory now returns the modified instance of tv4.
```
Comment (quantumew) [inline CHANGELOG.md]: Can you a bit more explicit? you are making this work nicer with Dizzy. Maybe put a comment about that.

**Review by quantumew — APPROVED** [review]
Two minor things then I approve

File: CHANGELOG.md (hunk 2 above)
Comment (GoogilyBoogily) [reply CHANGELOG.md]: Good call. Fixed f9edb39.

File: README.md (hunk 1 above)
Comment (GoogilyBoogily) [reply README.md]: Updated f9edb39.

**Pull request merged**